cuesubmit/cuesubmit/Submission.py

```diff
@@ -72,8 +74,19 @@ def buildBlenderCmd(layerData):
         renderCommand += ' -o {}'.format(outputPath)
     if outputFormat:
         renderCommand += ' -F {}'.format(outputFormat)
-    # The render frame must come after the scene and output
-    renderCommand += ' -f {frameToken}'.format(frameToken=Constants.FRAME_TOKEN)
+    if frameRange:
+        # Checks if frame range is in the correct format
+        if re.match(r'^\d+$', frameRange):
+            renderCommand += ' -f {}'.format(frameRange)
+        elif re.match(r'^\d+-\d+$', frameRange):
+            startFrame, endFrame = map(int, frameRange.split("-"))
+            renderCommand += (' -s {startFrame} -e {endFrame} -a'
```

Comment:
If I'm reading this correctly, I think this will result in a layer command with the start/end frames hardcoded. I.e., if the frame range is `1-5` the command will contain ` -s 1 -e 5 -a`.

Compare this to the code prior to this change, which instead inserts the `#IFRAME#` token to act as a placeholder for the frame.

I think instead, we want to use tokens for start frame / end frame: https://github.com/AcademySoftwareFoundation/OpenCue/blob/5b9defd5f5fa87b49270bf7b5851f84066952ed6/cuesubmit/cuesubmit/Constants.py#L45-L54

These tokens should allow Cuebot to swap in the correct frame numbers for each task based on the chunk size.

Reply:
@bcipriano noted. 
I was under the impression that the **Frame Spec** field solely defined the frame range (1-5), hence why I got it directly from that but had no idea about the **Chunk Size**.

> ...should allow Cuebot to swap in the correct frame numbers for each task based on the chunk size.

Got it. 
To make sure I follow correctly; **Frame Spec** can be populated with a frame range such as `1-5`, and then **Chunk Size** can be set as an integer value (eg: `2`) which defines the _number_ of frames _within_ that `1-5` range that will be processed per RQD node, yes?
And the dedicated #tokens can be used to substitute the respective values.

Sorry, I'm not very familiar with the concept of chunk sizes in render farms, hence the confusion 😅

Reply:
@bcipriano I've resolved this in https://github.com/AcademySoftwareFoundation/OpenCue/pull/1337/commits/ce618a05cef8d20cef62aee41ebde7fd2d7ea086.

Verified on test animation.
![image](https://github.com/AcademySoftwareFoundation/OpenCue/assets/24616975/5404e89c-51ea-43ad-b2d9-99d47c9b1288)